src/sidebar/components/LaunchErrorPanel.js

```diff
@@ -0,0 +1,24 @@
+import Panel from './Panel';
+
+/**
+ * @typedef LaunchErrorPanelProps
+ * @prop {Error} error - The error that prevented the client from launching
+ */
+
+/**
+ * An error panel displayed when a fatal error occurs during app startup.
+ *
+ * Note that this component cannot use any of the services or store that are
+ * normally available to UI components in the client.
+ *
+ * @param {LaunchErrorPanelProps} props
+ */
+export default function LaunchErrorPanel({ error }) {
+  return (
+    <div className="LaunchErrorPanel">
+      <Panel title="Unable to start Hypothesis">
```

Comment:
You could use the `restricted` icon on this panel (if icons are registered at this point?) to match other error panels like the direct-linked errors...

Reply:
Mmm ... I'm not sure that would give the right vibe. To me that suggests "not allowed" rather than "oh no, something went wrong. Try again!". Happy to discuss more on Slack.